spec/lib/spree/core/environment_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Spree::Core::Environment do
+  # Our version doesn't add any features we could test.
+  # So we just check that our file is loaded correctly.
```

Comment:
:+1:
Nice one.
I think I moved the file to remove the stock_splitters attribute.